src/cardboard-vr-display.js

```diff
@@ -136,7 +138,7 @@ CardboardVRDisplay.prototype.beginPresent_ = function() {
     this.exitPresent();
```

Comment:
should handle https://github.com/borismus/webvr-polyfill/pull/97/files#diff-55f22139cda256d0c2e29cb114cdf4f0R121 and https://github.com/borismus/webvr-polyfill/pull/97/files#diff-55f22139cda256d0c2e29cb114cdf4f0R133 too